container-search/src/main/java/com/yahoo/search/rendering/JsonRenderer.java

```diff
@@ -788,11 +642,11 @@ protected void renderFieldContents(Object field) throws IOException {
             if (field instanceof Inspectable && ! (field instanceof FeatureData)) {
                 renderInspector(((Inspectable)field).inspect());
             } else {
-                renderFieldContentsDirect(field);
+                consume(field);
             }
         }
 
-        private void renderFieldContentsDirect(Object field) throws IOException {
+        public void consume(Object field) throws IOException {
```

Comment:
Isn't the naming convention here "acceptX"?

Reply:
Correct -> fixed